src/audio/commands/Audio/exportqueue.ts

```diff
@@ -4,8 +4,10 @@ import type { GuildMessage } from '#lib/types/Discord';
 import { map, prependIfNotNull, take } from '#utils/common';
 import { getAudio } from '#utils/functions';
 import { ApplyOptions } from '@sapphire/decorators';
+import { CommandOptionsRunTypeEnum } from '@sapphire/framework';
 import { send } from '@sapphire/plugin-editable-commands';
 import { serialize } from 'binarytf';
+import { PermissionFlagsBits } from 'discord-api-types/payloads/v9';
```

Comment:
That's not the import it should've suggested...

Reply:
It suggested a whole bunch :notlikethis: